packages/mwp-i18n/src/withIntl.jsx

```diff
@@ -2,9 +2,18 @@
 import React from 'react';
 import type { ComponentType } from 'react';
 import { IntlProvider, injectIntl } from 'react-intl';
+import { connect } from 'react-redux';
+import type { MapStateToProps } from 'react-redux';
 
-type Messages = { [string]: string };
-type Props = { __locale?: string, [string]: any };
+const DEFAULT_LOCALE = 'en-US';
+type TRNSource = { [string]: string };
+// Messages must support 'en-US'
+type Messages = { 'en-US': TRNSource, [string]: TRNSource };
```

Comment:
maybe i'm slightly confused about the changes, but this seems to always expect an object organized by locale of trn messages... 
```js
{ LOCALE: [... MESSAGES OBJECT...] }
```

Reply:
yes indeed - that's the new format produced by the mwp-cli PR https://github.com/meetup/mwp-cli/pull/49

Reply:
Ahhhhh! I missed that detail in the previous PR.